front/lib/api/files/upsert.ts

```diff
@@ -230,7 +237,13 @@ const upsertExcelToDatasource: ProcessingFunction = async (
   let worksheetContent: string | undefined;
 
   if (upsertArgs && !isUpsertTableArgs(upsertArgs)) {
-    return new Err(new Error("Invalid upsert args"));
+    return new Err<DustError>({
```

Comment:
I think this should be an `internal_error` here we should never land here so no need to cut a new code for it.